### ISSUE TITLE
Wrong CSV file naming if resample base period is included in file naming

### ISSUE THREAD

**Issue by nilskujath** [nilskujath/sttk#1]
## Concerned Code

This concerns the `download_and_resample_yf_data` function in the `sttk.data.yfdownload` module.

## Description

If the `include_resample_base_period` parameter is set to `True`, the name of the output CSV file is correct: 
```
>>> yfd.download_and_resample_yf_data(
...     include_resample_base_period=True,
...     period = '2d',
...     data_dir = '.',
... )
[*********************100%%**********************]  1 of 1 completed
Data successfully saved to ./sttk-data/AAPL_historical_2023-01-01_2023-12-31_2d_from_1d.csv
```

However, when set to `False` (default), the data in the CSV corresponds to the resampled period length but the CSV file bears the wrong name (here 1d instead of 2d):
```
>>> yfd.download_and_resample_yf_data(
...     period = '2d',
...     data_dir = '.', 
... )
[*********************100%%**********************]  1 of 1 completed
Data successfully saved to ./sttk-data/AAPL_historical_2023-01-01_2023-12-31_1d.csv
```

**Comment by nilskujath** [comment]
Issue fixed with commit e8151e0.

**Comment by nilskujath** [comment]
Issue fixed with commit e8151e0.
